plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -501,6 +501,95 @@ void GSRendererDX11::EmulateChannelShuffle(GSTexture** rt, const GSTextureCache:
 	}
 }
 
+void GSRendererDX11::EmulateBlending()
+{
+	// Partial port of OGL SW blending. Currently only works for accumulation blend.
+	const GIFRegALPHA& ALPHA = m_context->ALPHA;
+	bool sw_blending         = false;
+
+	// No blending so early exit
+	if (!(PRIM->ABE || (PRIM->AA1 && m_vt.m_primclass == GS_LINE_CLASS)))
+		return;
+
+	m_om_bsel.abe = 1;
+	m_om_bsel.a = ALPHA.A;
+	m_om_bsel.b = ALPHA.B;
+	m_om_bsel.c = ALPHA.C;
+	m_om_bsel.d = ALPHA.D;
+
+	if (m_env.PABE.PABE)
+	{
+		if (m_om_bsel.a == 0 && m_om_bsel.b == 1 && m_om_bsel.c == 0 && m_om_bsel.d == 1)
+		{
+			// this works because with PABE alpha blending is on when alpha >= 0x80, but since the pixel shader
+			// cannot output anything over 0x80 (== 1.0) blending with 0x80 or turning it off gives the same result
+
+			m_om_bsel.abe = 0;
+		}
+		else
+		{
+			//Breath of Fire Dragon Quarter triggers this in battles. Graphics are fine though.
+			//ASSERT(0);
+		}
+	}
+
+	uint8 blend_index  = uint8(((ALPHA.A * 3 + ALPHA.B) * 3 + ALPHA.C) * 3 + ALPHA.D);
+	int blend_flag = m_dev->GetBlendFlags(blend_index);
+
+	bool accumulation_blend = !!(blend_flag & BLEND_ACCU);
+
+	switch (m_sw_blending)
+	{
+	case ACC_BLEND_HIGH_D3D11:
+	case ACC_BLEND_MEDIUM_D3D11:
+	case ACC_BLEND_BASIC_D3D11:
+		sw_blending |= accumulation_blend;
+	default: break;
+	}
+
+	if (m_env.COLCLAMP.CLAMP == 0)
+		m_ps_sel.hdr = 1;
```

Comment:
Shouldn't it be 
```
	if (m_env.COLCLAMP.CLAMP == 0)
	{
		if (accumulation_blend)
			sw_blending = true;
		m_ps_sel.hdr = 1;
	}
```

Reply:
Hmm, ok this will probably be better/more clear in case there are changes to SW blending later on. I had left it out because after I removed the other SW blending we actually have sw_blending == accumulation_blend (if basic or higher mode) since accu blend is the only sw blend.